com.dz4k.FruitCredits.json

```diff
@@ -0,0 +1,3397 @@
+{
+  "id": "com.dz4k.FruitCredits",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.vala"
+  ],
+  "command": "fruit-credits",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--env=LD_LIBRARY_PATH=/app/lib:/app/lib64:/usr/lib/x86_64-linux-gnu/GL/default/lib:/usr/lib/x86_64-linux-gnu/openh264/extra:/usr/lib/x86_64-linux-gnu/openh264/extra:/usr/lib/sdk/vala/lib:/usr/lib/x86_64-linux-gnu/GL/default/lib"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/vala/bin",
+    "prepend-ld-library-path": "/usr/lib/sdk/vala/lib",
+    "env": {}
+  },
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "/share/vala",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "ghc-9.6.6-x86_64",
+      "only-arches": [
+        "x86_64"
+      ],
+      "cleanup": [
+        "/bin/ghc",
+        "/bin/ghc-9.6.6",
+        "/bin/ghc-pkg",
+        "/bin/ghc-pkg-9.6.6",
+        "/bin/ghci",
+        "/bin/ghci-9.6.6",
+        "/bin/haddock",
+        "/bin/haddock-ghc-9.6.6",
+        "/bin/runghc",
+        "/bin/runghc-9.6.6",
+        "/bin/runhaskell",
+        "/bin/hp2ps",
+        "/bin/hpc",
+        "/bin/hsc2hs",
+        "/lib/libtinfo.so.5",
+        "/lib/debug",
+        "/lib/ghc-9.6.6",
+        "/share/man",
+        "/share/doc/ghc-9.6.6"
+      ],
+      "buildsystem": "simple",
+      "builddir": false,
+      "build-commands": [
+        "mkdir -p /app/lib",
+        "ln -s /usr/lib/x86_64-linux-gnu/libtinfo.so /app/lib/libtinfo.so.5",
+        "./configure --prefix=/app",
+        "make install"
+      ],
+      "sources": [
+        {
+          "sha256": "ff5b4929a4e89c536e7badb3b142e353dc4bda1f31d3ee446406ad88c3bebddf",
+          "type": "archive",
+          "url": "https://downloads.haskell.org/ghc/9.6.6/ghc-9.6.6-x86_64-deb9-linux.tar.xz"
```

Comment:
how will this work on aarch64?

Reply:
It won't, sorry. The tool I used to generate the haskell modules has a bug relating to aarch64, which I didn't initially notice. I've added an aarch64 ghc module now.